### PR TITLE
docs: release note 1.1.3

### PR DESCRIPTION
**Pull request by AnthonyLaw**
PR all merged, release to merge.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,22 @@ All notable changes to this project will be documented in this file.
...
+SDK Core| v1.0.3 | [symbol-sdk][symbol-sdk@v1.0.3]
+
+### Added
+- Support new transaction type (revokable) [#959](https://github.com/symbol/symbol-explorer/issues/959)
+- Testnet theme [#971](https://github.com/symbol/symbol-explorer/pull/971)
+
+### Fixed
+- Invalid prop: type check failed for prop "transactionType" [#963](https://github.com/symbol/symbol-explorer/issues/963)
+- Unknown custom element [#964](https://github.com/symbol/symbol-explorer/issues/964)
+- Partial transaction can not view in Explorer [#972](https://github.com/symbol/symbol-explorer/issues/972)
```
Comment (segfaultxavi) [inline CHANGELOG.md:20]:
```suggestion
- Partial transactions cannot be viewed in Explorer [#972](https://github.com/symbol/symbol-explorer/issues/972)
```

File: README.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ Copyright 2019-present NEM
...
 Licensed under the [Apache License 2.0](LICENSE)
 
-[self]: https://github.com/nemgrouplimited/symbol-explorer
-[docs]: https://nemtech.github.io
-[issues]: https://github.com/nemgrouplimited/symbol-explorer/issues
-[slack]: https://join.slack.com/t/nem2/shared_invite/enQtMzY4MDc2NTg0ODgyLWZmZWRiMjViYTVhZjEzOTA0MzUyMTA1NTA5OWQ0MWUzNTA4NjM5OTJhOGViOTBhNjkxYWVhMWRiZDRkOTE0YmU
+[self]: https://github.com/symbol/symbol-explorer
+[docs]: https://docs.symbolplatform.com
+[issues]: https://github.com/symbol/symbol-explorer/issues
+[discord]: https://discord.com/invite/
```
Comment (segfaultxavi) [inline README.md:67]:
```suggestion
[discord]: https://discord.com/invite/xymcity
```

**Review by segfaultxavi — CHANGES_REQUESTED** [review]

**Review by segfaultxavi — APPROVED** [review]

**Review by segfaultxavi — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -406,23 +433,27 @@ SDK Core| v0.20.7 | [symbol-sdk](https://www.npmjs.com/package/symbol-sdk)
...
 [symbol-sdk@v1.0.1]: https://www.npmjs.com/package/symbol-sdk/v/1.0.1
 [symbol-sdk@v1.0.2]: https://www.npmjs.com/package/symbol-sdk/v/1.0.2
+[symbol-sdk@v1.0.3]: https://www.npmjs.com/package/symbol-sdk/v/1.0.3
 
 [catapult-rest@v2.3.0]: https://github.com/nemtech/catapult-rest/releases/tag/v2.3.0
 [catapult-rest@v2.3.3]: https://github.com/nemtech/catapult-rest/releases/tag/v2.3.3
 [catapult-rest@v2.3.4]: https://github.com/nemtech/catapult-rest/releases/tag/v2.3.4
 [catapult-rest@v2.3.6]: https://github.com/nemtech/catapult-rest/releases/tag/v2.3.6
 [catapult-rest@v2.3.7]: https://github.com/symbol/catapult-rest/releases/tag/v2.3.7
 [catapult-rest@v2.3.8]: https://github.com/symbol/catapult-rest/releases/tag/v2.3.8
```
Comment (Jaguar0625) [inline CHANGELOG.md:458]: i don't think you need to show diffs for any of dependencies. just diffs for _this_ project.

Comment (AnthonyLaw) [reply CHANGELOG.md:458]: right, i will remove all of this 👍🏼

**Review by Jaguar0625 — DISMISSED** [review]

**Review by Jaguar0625 — APPROVED** [review]
please remember to squash this before merging

**Pull request merged**